test/unit/auth/token-generator.spec.ts

```diff
@@ -293,22 +294,22 @@ describe('FirebaseTokenGenerator', () => {
     it('should throw given no uid', () => {
       expect(() => {
         (tokenGenerator as any).createCustomToken();
-      }).to.throw('First argument to createCustomToken() must be a non-empty string uid');
+      }).to.throw(FirebaseAuthError).with.property('code', 'auth/argument-error');
```

Comment:
I've switched many of the error check tests from 'verify the error message' to 'verify the error code'. Personally, I prefer just verifying the code, as the error message itself often changes which makes the tests a little fragile. But this is hardly a universal opinion. I'm happy to add the error message back in if you'd prefer.

(Obviously, these changes in particular were caused by me tweaking the error message slightly and having a bunch of tests fail as a result.)